openapi/openapi/spec3.json

```diff
@@ -3391,21 +3291,21 @@
                 "items": {
                   "type": "string",
                   "format": "address",
-                  "description": "Receiving address (+E.164 formatted phone number)"
+                  "description": "translation missing: en.api.desc.message.to.short_code"
```

Comment:
Need to add this?

Reply:
Yep, I'll make that change. For the mock and update the docs.

Reply:
Was this from a problem in the Ruby docs -> openapi-internal -> here?